tests/integration/istioio/README.md

```diff
@@ -146,35 +140,126 @@ istioio.IfMinikube{
 }
 ```
 
-## Run Shell Commands
+## Running Shell Commands
 
-You can create a test step that will run a shell command with `istioio.Command`:
+You can create a test step that will run a bash script and automatically generate
```

Comment:
Do we still have zsh and other shells discussed in the Istio? I know we did at some point.

Reply:
Nit, just thinking about how we're specifying bash.

Reply:
Looks like Kiali and auto completion talk about it.